app/bundles/CoreBundle/Security/Cryptography/Cipher/Symmetric/McryptCipher.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * @copyright   2017 Mautic Contributors. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Security\Cryptography\Cipher\Symmetric;
+
+use Mautic\CoreBundle\Security\Exception\Cryptography\Symmetric\InvalidDecryptionException;
+
+/**
+ * Class McryptCryptography.
+ *
+ * @deprecated
```

Comment:
Could you provide info which class should be used instead? It could be useful for devs.